### PR TITLE
fix: Avatar group responsive font not working

### PR DESCRIPTION
**Pull request by laishiwen**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fonts inside Avatar.Group components support responsive |
| 🇨🇳 Chinese | Avatar.Group 组件内的字体支持响应式 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by laishiwen** [comment]
1、Avatar group font-size is invalid in responsive；
2、Font size inheritance private variable is more elegant；

**Comment by laishiwen** [comment]
@zombieJ code review

**Comment by laishiwen** [comment]
屏幕录制2022-04-01 下午11.47.27

File: components/avatar/style/index.less [hunk 1]
```diff
@@ -5,11 +5,11 @@
 
 .@{avatar-prefix-cls} {
   .reset-component();
-
   position: relative;
   display: inline-block;
   overflow: hidden;
   color: @avatar-color;
+  font-size: var(--fontSize, inherit);
```
Comment (yoyo837) [inline components/avatar/style/index.less:12]: 不支持IE呀。

File: components/avatar/style/index.less [hunk 2]
```diff
@@ -5,7 +5,6 @@
 
 .@{avatar-prefix-cls} {
   .reset-component();
```
Comment (afc163) [inline components/avatar/style/index.less:7]:
```suggestion
  .reset-component();

```

**Review by afc163 — APPROVED** [review]

**Comment by yoyo837** [comment]
CI failed.

**Pull request merged**